packages/discovery-provider/src/api/v1/users.py

```diff
@@ -3007,3 +3011,160 @@ def get(self, id, authed_user_id):
         muted_users = get_muted_users(decoded_id)
         muted_users = list(map(extend_user, muted_users))
         return success_response(muted_users)
+
+
+encrypted_email_response = make_response(
+    "encrypted_email_response", ns, fields.List(fields.Nested(encrypted_email))
+)
+
+USER_ENCRYPTED_EMAILS_ROUTE = "/<string:id>/encrypted_emails"
```

Comment:
given that we don't really have a "non-encrypted" email and it's also not exactly following the convention of "belongs to" (in that this is not the email(s) for this user, but the emails the user has access to)

maybe `/id/shared_emails` ? or even `/id/shared/emails` ? wdyt?

Reply:
maybe going more abstract is actually better even: `users/<id>/fans` or `users/<id>/contacts` - yes the data includes an encrypted email but that's not really what they are in relation to this user?